builtin/credential/jwt/path_cel_role.go

```diff
@@ -0,0 +1,485 @@
+// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
+// SPDX-License-Identifier: MPL-2.0
+
+package jwtauth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-jose/go-jose/v3/jwt"
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/checker/decls"
+	"github.com/openbao/openbao/sdk/v2/framework"
+	celhelper "github.com/openbao/openbao/sdk/v2/helper/cel"
+	"github.com/openbao/openbao/sdk/v2/logical"
+	"github.com/openbao/openbao/sdk/v2/plugin/pb"
+)
+
+type celRoleEntry struct {
+	Name          string               `json:"name"`                     // Required
+	CelProgram    celhelper.CelProgram `json:"cel_program"`              // Required
+	FailurePolicy string               `json:"failure_policy,omitempty"` // Default: Deny
```

Comment:
If I'm not mistaken, isn't `FailurePolicy` unused? I don't know that it has a role under the single `CelProgram` design, tbh.

Reply:
I'll remove -- I think that was left over from adapting `Role`. Now, we could return Auth with failure policy from the CEL expression.

Reply:
Thank you!